style.css

```diff
@@ -369,6 +369,185 @@ button:active {
   background: var(--colorButtonPressed);
 }
 
+/* ----- POPUP CREATION ----- */
+
+.popup__window {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--backgroundColor);
+  background-blend-mode: multiply;
+  z-index: 100;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
```

Comment:
Great job @mateoburvi 🤝

- [ ] As requested by the previous reviewer [here](https://github.com/mateoburvi/Portfolio/pull/12#discussion_r1249247967), kindly make sure to place the close button in accordance with the given Figma template✨⛏ to get a design close to the Figma template as much as possible ✌